### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by computeronix**
Modified to include automatic instructions

Feel free to clean up as needed

### CODE REVIEW

**Comment by bookchin** [comment]
Can you use [markdown syntax](https://daringfireball.net/projects/markdown/basics) please?

**Comment by computeronix** [comment]
Updated

File: README.md [hunk 1]
```diff
@@ -40,6 +40,19 @@ nvm install 4.4.4
...
+ 1.) Download Latest Release of storj-automation (`https://github.com/Storj/storj-automation/archive/master.zip`) 
+ 2.) Extract ZIP, and navigate to `storj-automation-master\windows\storjshare-cli-automate`
+ 3.) Double-click `install.bat`
+ 4.) (if prompted) Click Yes on the User Account Control (UAC) screen 
+ 5.) (if applicable) Reboot when completed
+ 6.) Double-click `install.bat`
+ 7.) Installation should be completed. Follow `https://github.com/Storj/storjshare-cli#usage` to complete.
+```
+
+Manual
```
Comment (bookchin) [inline README.md]: Change this to:

```
#### Manual
```

So it renders like:

#### Manual

File: README.md [hunk 2]
```diff
@@ -40,6 +40,19 @@ nvm install 4.4.4
...
+Automated Script
+```
+ 1.) Download Latest Release of storj-automation (`https://github.com/Storj/storj-automation/archive/master.zip`) 
+ 2.) Extract ZIP, and navigate to `storj-automation-master\windows\storjshare-cli-automate`
+ 3.) Double-click `install.bat`
+ 4.) (if prompted) Click Yes on the User Account Control (UAC) screen 
+ 5.) (if applicable) Reboot when completed
+ 6.) Double-click `install.bat`
+ 7.) Installation should be completed. Follow `https://github.com/Storj/storjshare-cli#usage` to complete.
+```
```
Comment (bookchin) [inline README.md]: This block can actually just be a regular ordered list. Change to:

```
1. Download Latest Release of storj-automation (`https://github.com/Storj/storj-automation/archive/master.zip`) 
2. Extract ZIP, and navigate to `storj-automation-master\windows\storjshare-cli-automate`
3. Double-click `install.bat`
4. (if prompted) Click Yes on the User Account Control (UAC) screen 
5. (if applicable) Reboot when completed
6. Double-click `install.bat`
7. Installation should be completed. Follow `https://github.com/Storj/storjshare-cli#usage` to complete.
```

So it renders like:

1. Download Latest Release of storj-automation (`https://github.com/Storj/storj-automation/archive/master.zip`) 
2. Extract ZIP, and navigate to `storj-automation-master\windows\storjshare-cli-automate`
3. Double-click `install.bat`
4. (if prompted) Click Yes on the User Account Control (UAC) screen 
5. (if applicable) Reboot when completed
6. Double-click `install.bat`
7. Installation should be completed. Follow `https://github.com/Storj/storjshare-cli#usage` to complete.

**Comment by bookchin** [comment]
Sorry it's taken until now to get back on this. See my comments (sorry if it seems nitpicky).

**Comment by computeronix** [comment]
no problem -- updated

even made the links clickable :)

**Comment by bookchin** [comment]
nice, thanks!

**Pull request merged**